OpenSim/Simulation/Manager/Manager.h

```diff
@@ -76,7 +76,9 @@ class OSIMSIMULATION_API Manager
     /** Simulation session name. */
     std::string _sessionName;
     /** Model for which the simulation is performed. */
-    Model *_model;
+    Model* _model;
+    /** Keep a pointer to the State passed in so we can update it. */
+    SimTK::State* _state;
```

Comment:
`ReferencePtr` here too.

Reply:
+1, but it seems like storing this will no longer be necessary (see other comments).